sections/leaderboard/TraderHistory.tsx

```diff
@@ -0,0 +1,398 @@
+									},
+									{
+										Header: (
+											<TableHeader>{t('leaderboard.trader-history.table.total-pnl')}</TableHeader>
+										),
+										accessor: 'pnl',
+										Cell: (cellProps: CellProps<any>) => (
+											<PnlContainer>
+												<ColorCodedPrice
+													currencyKey={'sUSD'}
+													price={cellProps.row.original.pnl}
+													sign={'$'}
+													conversionRate={1}
+												/>
+												<StyledValue
+													color={
+														cellProps.row.original.pnl.gt(0)
+															? 'green'
+															: cellProps.row.original.pnl.lt(0)
+															? 'red'
+															: ''
+													}
+												>
+													{cellProps.row.original.pnlPct}
+												</StyledValue>
+											</PnlContainer>
+										),
+										width: 40,
+										sortType: 'basic',
+										sortable: true,
+									},
+								],
+							},
+						]}
+					/>
+				</MobileOrTabletView>
+			</>
+		);
+	}
+);
+
+const StyledTable = styled(Table)<{ compact: boolean | undefined }>`
+	margin-top: ${({ compact }) => (compact ? '0' : '15px')};
+`;
+
+const TableTitle = styled.div`
+	width: 100%;
+	display: flex;
+	justify-content: flex-start;
+`;
+
+const TitleText = styled.a`
+	font-family: ${(props) => props.theme.fonts.regular};
+	color: ${(props) => props.theme.colors.selectedTheme.gray};
+
+	&:hover {
+		text-decoration: underline;
+		cursor: pointer;
+	}
+`;
+
+const StyledCell = styled.div`
+	color: ${(props) => props.theme.colors.selectedTheme.button.text.primary};
+	display: flex;
+`;
+
+const TitleSeparator = styled.div`
+	margin-left: 10px;
+	margin-right: 10px;
+	font-family: ${(props) => props.theme.fonts.regular};
+	color: ${(props) => props.theme.colors.selectedTheme.gray};
+`;
+
+const TraderText = styled.a`
+	font-family: ${(props) => props.theme.fonts.regular};
+	color: ${(props) => props.theme.colors.selectedTheme.button.text.primary};
+
+	&:hover {
+		text-decoration: underline;
+	}
+`;
+
+const TableHeader = styled.div`
+	font-family: ${(props) => props.theme.fonts.regular};
+	color: ${(props) => props.theme.colors.selectedTheme.gray};
+`;
+
+const StyledCurrencyIcon = styled(CurrencyIcon)`
+	width: 30px;
+	height: 30px;
+	margin-right: 5px;
+`;
+
+const CurrencyInfo = styled(FlexDiv)`
+	align-items: center;
+`;
+
+const StyledSubtitle = styled.div`
+	font-family: ${(props) => props.theme.fonts.regular};
+	font-size: 13px;
+	color: ${(props) => props.theme.colors.selectedTheme.button.text.primary};
+	text-transform: capitalize;
+`;
+
+const PnlContainer = styled.div`
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+`;
+
+const ColorCodedPrice = styled(Currency.Price)`
+	align-items: right;
+	color: ${(props) =>
+		props.price > 0
+			? props.theme.colors.selectedTheme.green
+			: props.price < 0
+			? props.theme.colors.selectedTheme.red
+			: props.theme.colors.selectedTheme.button.text.primary};
+`;
+
+const StyledValue = styled.div`
```

Comment:
Seems we could extend the component above here as similar color logic, maybe change price prop to more generic `value` so both price and amount can be passed to determine color.

Reply:
Done. Pushed in the latest commit.